docusaurus/sidebars.js

```diff
@@ -562,6 +562,8 @@ module.exports = {
         type: "generated-index",
       },
       items: [
+        "release_notes/october_2023",
+        "release_notes/september_2023",
```

Comment:
Would it make sense to to keep the migration notes on the top here?

Reply:
Moved!